documentation/src/main/asciidoc/topics/contributing/ui.adoc

```diff
@@ -0,0 +1,29 @@
+[[design_guidelines]]
+= UI Writing Guidelines
+
+This section describes guidelines for working with text in {brandname} user
+interface (UI) elements and applies, but is not limited, to the following
+components:
+
+* {brandname} Console
+* {brandname} CLI
+* log messages
+* error messages
+
+Content in the {brandname} UI should adhere to the link:https://www.patternfly.org/v4/ux-writing/about[PatternFly content style guide].
+
+While the PatternFly design guidelines are the definitive resource for UI writing guidelines, here are some rules for creating microcopy in {brandname} UI:
+
+* Headings on UI elements use sentence casing. link:https://www.patternfly.org/v4/ux-writing/capitalization/[PatternFly capitalization rules]
+* Dialogs that prompt for user action should have the following structure:
++
+Heading in question form, starting with verb. ("Create new entry?") +
+Body that provides details and outcome of action. +
+Body that uses a synonym to the verb in heading where ambiguity might exist. +
+Buttons.
+* Use past tense form for success or failure messages.
++
+*Yes:* "Cache created." *No:* "Cache has been created."
+* Avoid using the word "please".
+* Ensure all icons and buttons with action have tooltip/hover help.
+* Avoid abbreviations such as "avg" instead of "average".
```

Comment:
We might also want to add "No." here since that appeared a lot. We should also avoid "e.g." and "i.e." Those sneak into UIs sometimes, but they're not easily understood by all users (especially those who speak a language without Latin roots). So maybe we could just expand this by saying something like, 

_Only use common and familiar abbreviations, such as "URL." Write out all other abbreviations, especially the following:_ 

_- Number: Avoid "No."_ 
_- Average: Avoid "avg."_ 
_- For example: Avoid "e.g."_ 
_- In other words: Avoid "i.e."_  

_For more information on abbreviations and acronyms, see [PatternFly's Numerics page in the UX writing style guide.](https://www.patternfly.org/v4/ux-writing/terminology)_